--- CMakeLists.txt
+++ CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.4.3)
+cmake_minimum_required(VERSION 2.8.12)
 set(CMAKE_ALLOW_LOOSE_LOOP_CONSTRUCTS true)
 cmake_policy(SET CMP0017 NEW)
 #this line has to appear before 'PROJECT' in order to be able to disable incremental linking
@@ -45,14 +45,14 @@
 IF(NOT WIN32)
 	SET(DL ${CMAKE_DL_LIBS})
 	IF(CMAKE_SYSTEM_NAME MATCHES "Linux")
-		MESSAGE("Linux")
+		#MESSAGE("Linux")
 		SET(OSDEF -D_LINUX)
 	ELSE(CMAKE_SYSTEM_NAME MATCHES "Linux")
 		IF(APPLE)
-			MESSAGE("Apple")
+			#MESSAGE("Apple")
 			SET(OSDEF -D_DARWIN)
 		ELSE(APPLE)
-			MESSAGE("BSD?")
+			#MESSAGE("BSD?")
 			SET(OSDEF -D_BSD)
 		ENDIF(APPLE)
 	ENDIF(CMAKE_SYSTEM_NAME MATCHES "Linux")
@@ -309,6 +309,7 @@
 # This is the shortcut to finding GLU, GLUT and OpenGL if they are properly installed on your system
 # This should be the case.

+#[[
 FIND_PACKAGE(OpenGL)
 IF (OPENGL_FOUND)
 	MESSAGE("OPENGL FOUND")
@@ -318,6 +319,7 @@
 	SET(OPENGL_gl_LIBRARY opengl32)
 	SET(OPENGL_glu_LIBRARY glu32)
 ENDIF (OPENGL_FOUND)
+]]


 #FIND_PACKAGE(GLU)
